`runtime-coreclr r2r` pipeline failing to build

https://dev.azure.com/dnceng/public/_build/results?buildId=679965&view=logs&j=db768a9e-3325-5ce3-2d52-896c40bee68a&t=ea573249-0a9e-59d9-0a41-40ec8869f91f

Failure in "Build managed test components" step with lots of errors like this:
```
Building step 'Managed tests build (build tests)' testGroupToBuild=1 via "/Users/runner/runners/2.169.1/work/1/s/eng/common/msbuild.sh" --ci --warnAsError false /Users/runner/runners/2.169.1/work/1/s/src/coreclr/tests/build.proj /p:RestoreDefaultOptimizationDataPackage=false /p:PortableBuild=true /p:UsePartialNGENOptimization=false /maxcpucount "/flp:Verbosity=normal;LogFile=/Users/runner/runners/2.169.1/work/1/s/artifacts/log/Tests_Managed.OSX.x64.Checked.log;Append=false" "/flp1:WarningsOnly;LogFile=/Users/runner/runners/2.169.1/work/1/s/artifacts/log/Tests_Managed.OSX.x64.Checked.wrn;Append=false" "/flp2:ErrorsOnly;LogFile=/Users/runner/runners/2.169.1/work/1/s/artifacts/log/Tests_Managed.OSX.x64.Checked.err;Append=false"  /p:TargetArchitecture=x64 /p:Configuration=Checked /p:TargetOS=OSX /nodeReuse:false     /p:CLRTestBuildAllTargets=allTargets /p:CLRTestPriorityToBuild=1 "/p:CopyNativeProjectBinaries=false" /p:__SkipPackageRestore=true
Clearing NuGet HTTP cache: /Users/runner/.local/share/NuGet/v3-cache
Local resources cleared.
  Building Targeting Pack
  external -> 
/Users/runner/runners/2.169.1/work/1/s/eng/liveBuilds.targets(147,5): error : The 'libs' subset must be built before building this project. Missing artifacts: /Users/runner/runners/2.169.1/work/1/s/artifacts/bin/ref/microsoft.netcore.app/Release/ [/Users/runner/runners/2.169.1/work/1/s/src/coreclr/tests/src/Common/CoreCLRTestLibrary/CoreCLRTestLibrary.csproj]
##[error]eng/liveBuilds.targets(147,5): error : The 'libs' subset must be built before building this project. Missing artifacts: /Users/runner/runners/2.169.1/work/1/s/artifacts/bin/ref/microsoft.netcore.app/Release/
/Users/runner/runners/2.169.1/work/1/s/eng/liveBuilds.targets(147,5): error : The 'libs' subset must be built before building this project. Missing artifacts: /Users/runner/runners/2.169.1/work/1/s/artifacts/bin/ref/microsoft.netcore.app/Release/ [/Users/runner/runners/2.169.1/work/1/s/src/coreclr/tests/src/CoreMangLib/system/delegate/delegate/DelegateEquals1.csproj]
##[error]eng/liveBuilds.targets(147,5): error : The 'libs' subset must be built before building this project. Missing artifacts: /Users/runner/runners/2.169.1/work/1/s/artifacts/bin/ref/microsoft.netcore.app/Release/
```

Looks like it started with 20200604.1:
https://dev.azure.com/dnceng/public/_build/results?buildId=672785&view=results

@dotnet/runtime-infrastructure

--- a/eng/pipelines/coreclr/r2r-extra.yml
+++ b/eng/pipelines/coreclr/r2r-extra.yml
@@ -34,6 +34,7 @@ jobs:
     - CoreClrTestBuildHost # Either OSX_x64 or Linux_x64
     jobParameters:
       testGroup: r2r-extra
+      liveLibrariesBuildConfig: Release
 
 - template: /eng/pipelines/common/platform-matrix.yml
   parameters:

--- a/eng/pipelines/coreclr/r2r.yml
+++ b/eng/pipelines/coreclr/r2r.yml
@@ -40,6 +40,7 @@ jobs:
     - CoreClrTestBuildHost # Either OSX_x64 or Linux_x64
     jobParameters:
       testGroup: outerloop
+      liveLibrariesBuildConfig: Release
 
 - template: /eng/pipelines/common/platform-matrix.yml
   parameters:

